R/generate_cre_dataset.R

```diff
@@ -99,7 +97,7 @@ generate_cre_dataset <- function(n = 1000, rho = 0, n_rules = 2, p = 10,
 
   # Check for correct binary input
   if (!(binary_outcome %in% c(TRUE, FALSE))) {
-    stop("Invalid 'binary' input. Please specify TRUE or FALSE.")
+    stop("Invalid 'binary' input. Please specify `TRUE` or `FALSE`.")
```

Comment:
Please drop backticks from TRUE and FALSE in the code.

Reply:
Removed